gpii/node_modules/flowManager/test/CaptureTests.js

```diff
@@ -0,0 +1,328 @@
+/**
+ * GPII Capture Tests
+ *
+ * Copyright 2019 Raising the Floor - International
+ *
+ * Licensed under the New BSD license. You may not use this file except in
+ * compliance with this License.
+ *
+ * You may obtain a copy of the License at
+ * https://github.com/GPII/universal/blob/master/LICENSE.txt
+ */
+
+"use strict";
+
+var fluid = require("infusion"),
+    gpii = fluid.registerNamespace("gpii"),
+    kettle = require("kettle");
+
+var jqUnit = require("node-jqunit");
+
+fluid.require("%gpii-universal");
+gpii.loadTestingSupport();
+
+fluid.registerNamespace("gpii.tests.flowManager.capture");
+
+kettle.config.createDefaults({
+    configName: "gpii.flowManager.tests.capture.fakeData.config",
+    configPath: "%gpii-universal/gpii/node_modules/flowManager/test/configs"
+});
+
+fluid.defaults("gpii.tests.flowManager.capture.tests", {
+    gradeNames: ["fluid.test.testEnvironment", "fluid.test.testCaseHolder"],
+    modules: [
+        {
+            name: "Simple system capture",
+            tests: [{
+                name: "Check for existing FakeMag Settings",
+                expect: 2,
+                sequence: [{
+                    funcName: "gpii.tests.flowManager.capture.adjustSolutions",
+                    args: ["{config}.server.flowManager.solutionsRegistryDataSource"]
+                }, {
+                    task: "{config}.server.flowManager.capture.getSystemSettingsCapture",
+                    args: [],
+                    resolve: "gpii.tests.flowManager.capture.checkForFakeMags",
+                    resolveArgs: ["{arguments}.0"]
+                }]
+            }]
+        },
+        {
+            name: "Simple installed solutions fetch",
+            tests: [{
+                name: "Testing get installed solutions",
+                expect: 3,
+                sequence: [{
+                    funcName: "gpii.tests.flowManager.capture.adjustSolutions",
+                    args: ["{config}.server.flowManager.solutionsRegistryDataSource"]
+                }, {
+                    task: "{config}.server.flowManager.capture.getInstalledSolutions",
+                    args: [],
+                    resolve: "gpii.tests.flowManager.capture.checkForInstalledMags",
+                    resolveArgs: ["{arguments}.0"]
+                }]
+            }]
+        },
+        {
+            name: "Test Raw Settings Formatting",
```

Comment:
It would be easier to read to break this test for formatRawCapturedSettings(), as well as the test for filterInvokeSettingsHandlersReturn(), into their own jqUnit.test() rather than having one test stack with a few set of tests for unrelated functionalities. When writing tests for individual functions, you could define an object of test cases, each having an input and an expected output. jqUnit.test() can use a loop to loop thru this object. An example is [here](https://github.com/GPII/universal/blob/master/gpii/node_modules/flowManager/test/SystemUtilsTests.js#L118).

Reply:
I moved this test to it's own external jqUnit function, and put the functions that go with the test sequence further up closer in the file to them.